standalone/ansible/roles/sidekiq/templates/config/systemd/user/sidekiq@.service

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Sidekiq for simple-server {{ deploy_env }}
+After=syslog.target network.target
+
+# restarts/stops with sidekiq
+PartOf=sidekiq.service
+
+[Service]
+Type=simple
+Environment=RAILS_ENV=production
+WorkingDirectory=/home/deploy/apps/simple-server/current
+ExecStart=/home/deploy/.rbenv/shims/bundle exec sidekiq -e production
+ExecReload=/bin/kill -TSTP $MAINPID
+ExecStop=/bin/kill -TERM $MAINPID
+SyslogIdentifier=sidekiq
+
+# restart & stop send TERM to main process, wait up to 30 seconds, then KILL if still running
+KillMode=mixed
+TimeoutStopSec=30
+# restart on non-zero exit or other failure after 5 seconds
+Restart=on-failure
+RestartSec=5
+# don't create a new systemd.slice per instance
+Slice=system.slice
+
+[Install]
+WantedBy=default.target
```

Comment:
Could you remind me again why we're replacing `monit` with `systemd` to manage Sidekiq?

Reply:
We updated sidekiq to 6.0 recently, which discards PID file creation that monit used to track sidekiq. There are ways to make monit work but systemd is the right (and [recommended](https://github.com/mperham/sidekiq/wiki/Monitoring)) way to do it.

- Monit runs as root, which means we can't restart sidekiq as the deploy user. This interferes with our deploy script.
- We now have a [systemd config](https://github.com/simpledotorg/simple-server/pull/1014) in `simple-server` which spawns multiple sidekiq processes. We can't utilise that capability with `monit` in a straightforward manner.

Reply:
Thanks. I'm gonna leave this comment unresolved so we can find it later easily if needed.